Cargo.toml

```diff
@@ -2,13 +2,13 @@
 name = "gas-estimation"
 version = "0.1.0"
 edition = "2018"
-license = "MIT OR Apache-2.0"
 
 [dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
 futures = "0.3"
 primitive-types = { version = "0.10", features = ["fp-conversion"], optional = true }
+ethcontract = { git = "https://github.com/gnosis/ethcontract-rs.git", branch = "eip1559", features = ["gas_price"] }
```

Comment:
Maybe this should be `optional = true` so you have to build `gas-estimation` with `features = ["ethcontract"]` in order to include this dependency?